Reference/Searching/Examine/Quick-Start/index-v8.md

```diff
@@ -0,0 +1,260 @@
+            ? string.Empty
+            : Request["query"];
+        if (searchTerm == string.Empty)
+        {
+            <p>Enter search term</p>
+        }
+        else
+        {
+            //perform the search
+        }
+    }
+</div>
+```
+Here we are getting the request query from the form. If the string is empty we ask them to submit a term, otherwise we will perform the search.
+
+### Examine Search Index
+To perform the search we will first need to get a reference to the particular Examine index that we want to search. Then we will use this index to access it's corresponding `Searcher`. We use the `Searcher` to construct the query logic to execute and search the index.
+
+Umbraco ships with three indexes:
+* ExternalIndex - available to use for indexing published unprotected content.
+* InternalIndex - which Umbraco's backoffice search uses.
+* InternalMemberIndex - which Umbraco's Membership implementation uses.
+
+([You can create your own indexes too](../indexing)) if you need to analyse text in a different language for example.
+
+You use a convenient service named the `ExamineManager` to retrieve first the Index by its 'alias' and then use the Index to get a reference to the Searcher eg:
+```csharp
+ if(ExamineManager.Instance.TryGetIndex("ExternalIndex", out var index))
+    {
+        var searcher = index.GetSearcher();
+```
+### Creating the Search Query
+With this in mind we begin to update the `else` condition with the following:
+
+```csharp
+else
+{
+    //perform the search
+    //first we try to get the index, it is the ExternalIndex as we don't want to return unpublished things
+    //it returns the index in the var index
+    //be sure to add "@using Umbraco.Examine;" at the top of the view
+    if(ExamineManager.Instance.TryGetIndex("ExternalIndex", out var index))
+    {
+        var searcher = index.GetSearcher();
+        var results = searcher.CreateQuery("content").NodeTypeAlias("person").And().Field("nodeName", searchTerm).Execute();
+        if (results.Any())
+        {
+            <ul>
+                @foreach (var result in results)
+                {
+                    if (result.Id != null)
+                    {
+                        var node = Umbraco.Content(result.Id);
+                        <li>
+                            <a href="@node.Url()">@node.Name</a>
+                        </li>
+                    }
+                }
+            </ul>
+        }
+        else
+        {
+            <p>No results found for query @searchTerm</p>
+        }
+    }
+    return;
+}
+```
+At this point we have chosen to use the External index and it's searcher. 
+
+:::tip
+We reference the External index by it's alias "ExternalIndex". Umbraco has a set of 'Constants' that refer to the indexes that can be more convenient to use `Constants.UmbracoIndexes`. So, in the example here we could have used `Constants.UmbracoIndexes.ExternalIndexName` instead of "ExternalIndex".
```

Comment:
```suggestion
We reference the External index by its alias "ExternalIndex". Umbraco has a set of 'Constants' that refer to the indexes that can be more convenient to use `Constants.UmbracoIndexes`. So, in the example here we could have used `Constants.UmbracoIndexes.ExternalIndexName` instead of "ExternalIndex".
```